### PR TITLE
Improve test coverage expanders line graph generators solved (PR for issue #6034)

### PR DESCRIPTION
**Pull request by chimaobi-okite**
added tests to Improve test coverage on expanders line graph generators

### CODE REVIEW

**Comment by chimaobi-okite** [comment]
Please, can you review @MridulS

**Comment by MridulS** [comment]
Thanks for this @chimaobi-okite!

Please make the title of the pull request a bit more detailed, it's hard to understand "solved for issue" without searching for the issue. Something like "Improve test coverage on expanders line graph generators" can give reviewers a quick way of understanding the context of the changes in this pull request.

File: networkx/generators/tests/test_expanders.py [hunk 1]
```diff
@@ -70,3 +70,18 @@ def test_margulis_gabber_galil_graph_badinput(graph_type):
         nx.NetworkXError, match="`create_using` must be an undirected multigraph"
     ):
         nx.margulis_gabber_galil_graph(3, create_using=graph_type)
+
+
+@pytest.mark.parametrize("graph_type", (nx.Graph, nx.DiGraph))
+def test_chordal_cycle_graph_badinput(graph_type):
```
Comment (MridulS) [inline networkx/generators/tests/test_expanders.py:76]: This all looks good! Thanks @chimaobi-okite!

could you also add `nx.MultiDiGraph` ?

Comment (chimaobi-okite) [reply networkx/generators/tests/test_expanders.py:76]: Yes I can, I will do that now. Thank you!

**Comment by chimaobi-okite** [comment]
Hi @MridulS, your previous recommendations have been accounted for. Tests are also 100% covered.

**Comment by MridulS** [comment]
thanks for the contribution @chimaobi-okite!

**Pull request merged**